ts/features/fastLogin/store/reducers/sessionRefreshReducer.ts

```diff
@@ -44,17 +32,3 @@ const AutomaticSessionRefreshReducer = (
       return state;
   }
 };
-
-const CURRENT_REDUX_SESSION_REFRESH_STORE_VERSION = -1;
-
-const persistConfig: PersistConfig = {
-  key: "sessionRefresh",
-  storage: AsyncStorage,
-  version: CURRENT_REDUX_SESSION_REFRESH_STORE_VERSION,
-  whitelist: ["enabled"]
-};
-
-export const automaticSessionRefreshPersistor = persistReducer<
-  AutomaticSessionRefreshState,
-  Action
->(persistConfig, AutomaticSessionRefreshReducer);
```

Comment:
Usually, we prefer adding a migration when removing a previously persisted state to ensure a smooth transition for users across app versions. For example:

```ts
const CURRENT_REDUX_SESSION_REFRESH_STORE_VERSION = 0; // Increment version

const persistConfig: PersistConfig = {
  key: "sessionRefresh",
  storage: AsyncStorage,
  version: CURRENT_REDUX_SESSION_REFRESH_STORE_VERSION,
  migrate: Promise.resolve(_.omit(state, "enabled")), // Example, prefer using `createMigrations` instead
  whitelist: [] // No more keys to persist
};
```

This approach (or using `createMigrations`) allows us to manage the persisted state effectively throughout the app’s lifecycle while ensuring the correct state structure for users. We generally avoid removing the persistor entirely, as maintaining it helps with future flexibility and avoids potential issues with stale or unexpected data.

What do you think?

@LazyAfternoons, @Vangaorth what do you think about that?

Reply:
Thanks for this suggestion! I've add the migration!